justfile

```diff
@@ -80,6 +81,7 @@ docker-build-cli *ARGS:
 
 # Build the Support UI Docker image
 docker-build-ui *ARGS:
+  @cd {{solution-root}} && npm install -g sass
```

Comment:
I don't think we need this here.

Reply:
we do as the initial PR failed

Reply:
ID installs SASS explicitly before doing the build: https://github.com/DFE-Digital/get-an-identity/blob/main/.github/workflows/build.yml#L89

Reply:
Yeah but it does the build publish etc in the same workflow as the docker build - it is done in totally separate workflow in TRS in the package.yml

Reply:
I did add steps to install SASS in other parts of the build so I was also suprised it failed until I realised why!